### PR TITLE
Allow heads > 800m

### PR DESCRIPTION
**Pull request by elise-palethorpe**
Edit the double_half boolean to change between double half the head and linear interpolation

### CODE REVIEW

File: src/phes_base.cpp [hunk 1]
```diff
@@ -145,14 +145,51 @@ BigModel BigModel_init(GridSquare sc){
...
+	double energy_cost = dam_cost*1/(pair->water_rock*generation_efficiency * usable_volume*water_density*gravity*head)*J_GWh_conversion/cubic_metres_GL_conversion;
+	double power_cost;
+	double tunnel_cost;
+	double power_house_cost;
+	if (head > 800) {
+		if (double_half) {
+			power_house_cost = 2*calculate_power_house_cost(power/2, head/2);
+			tunnel_cost = 2*calculate_tunnel_cost(power/2, head/2, seperation);
+			power_cost = 0.001*(power_house_cost+tunnel_cost)/MIN(power, 800);
+		}
```
Comment (ryanstocks00) [inline src/phes_base.cpp]: Will double_half ever be true? Also, do we want to allow higher than 1600? e.g. an 1800m pair is modelled as 3 600m pairs?

Comment (ryanstocks00) [reply src/phes_base.cpp]: Ohhh, I just saw the comment in the PR, there should definitely be a comment in the code (and should probably be something more like `#DEFINE LINEAR_EXTRAPOLATE false` to make it clearer

Comment (elise-palethorpe) [reply src/phes_base.cpp]: Only if you change it in the file, should I make it a variable instead? I thought we should chat to Andrew before I went to that effort. Very happy to do up to 2400m - ask Andrew about that also?

Comment (ryanstocks00) [reply src/phes_base.cpp]: Do you want to shoot him an email and see?

Comment (elise-palethorpe) [reply src/phes_base.cpp]: Oops I missed your second comment, will do

**Review by ryanstocks00 — APPROVED** [review]

**Pull request merged**